addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.js

```diff
@@ -162,7 +162,12 @@ export class ProductLabelSectionAndNoteField extends Many2OneField {
     }
 
     get isProductClickable() {
-        return this.props.record.model.root.data.state !== "draft";
+        if (this.props.record.model.root.data) {
+            return this.props.record.model.root.data.state !== "draft";
+        }
+        else {
+            return this.props.record._parentRecord.data.state !== "draft";
+        }
```

Comment:
```suggestion
        if (this.props.record.model.root.data) {
            return this.props.record.model.root.data.state !== "draft";
        }
            
        return this.props.record._parentRecord.data.state !== "draft";
```
No need for the else, we can just use a guard clause.